src/openforms/forms/api/serializers/logic/action_serializers.py

```diff
@@ -225,4 +235,17 @@ def validate(self, data: dict) -> dict:
                 code="blank",
             )
 
+        # if action_type == LogicActionTypes.set_registration_backend:
+        #     if not action_value:
+        #         raise serializers.ValidationError(
+        #             {"action": {"value": _("Invalid registration backend")}},
+        #             code="blank",
+        #         )
+        #     # form is not in scope here :(
+        #     # if not form.registration_backends.filter(key=action_value).exists():
+        #     #     raise serializers.ValidationError(
+        #     #         {"action": {"value": _("Invalid registration backend")}},
+        #     #         code="invalid",
+        #     #     )
+        #
```

Comment:
have you tried getting it from `self.context["form"]` or adding it to the context if it's not there yet? But from `def logic_rules_bulk_update` it seems like it should be there.

Reply:
`form = self.context["form_variables"].form`, but turns out you already get a "invalid_choice" error as is.